x-pack/plugins/ml/readme.md

```diff
@@ -141,6 +141,19 @@ and Kibana instance that the tests will be run against.
 
     ML accessibility tests are located in `x-pack/test/accessibility/apps`.
 
+## Generating docs screenshots
+
+The screenshot generation uses the functional test runner described in the
+`Functional tests` section above.
+
+Run the following commands from the `x-pack` directory and use separate terminals
+for test server and test runner. The test server command starts an Elasticsearch
+and Kibana instance that the tests will be run against.
+
+    node scripts/functional_tests_server.js --config test/screenshot_creation/config.ts
+    node scripts/functional_test_runner.js --config test/screenshot_creation/config.ts --include-tag mlqa
+
+ML screenshot generation tests are located in `x-pack/test/screenshot_creation/apps/ml_docs`.
```

Comment:
Probably worth adding a line saying where the generated screenshots go `x-pack/test/functional/screenshots/session/ml_docs`

Reply:
Good idea! Done in a49e3eb.